windows-container-samples/msbuild/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM microsoft/windowsservercore
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop';"]
+
+RUN (New-Object System.Net.WebClient).DownloadFile('http://download.microsoft.com/download/5/f/7/5f7acaeb-8363-451f-9425-68a90f98b238/visualcppbuildtools_full.exe', 'visualcppbuildtools_full.exe') ; \
```

Comment:
Did you have another commit with that change? I want to get this merged but wasn't sure if you were going to switch back to Invoke-WebRequest or not